bucket/windowsdesktop-runtime.json

```
@@ -1,9 +1,9 @@
 {
     "version": "8.0.0",
-    "description": "Microsoft .NET 7.0 Desktop Runtime",
+    "description": "Microsoft .NET 8.0 Desktop Runtime",
     "homepage": "https://dotnet.microsoft.com/download/dotnet",
     "license": "MIT",
-    "notes": "You can now remove this installer with 'scoop uninstall windowsdesktop-runtime'",
+    "notes": "You can now remove this installer with 'scoop uninstall -p windowsdesktop-runtime'",
```

```suggestion
    "notes": "You can now remove this installer with 'scoop uninstall windowsdesktop-runtime'",
```
No persistence data has been created.